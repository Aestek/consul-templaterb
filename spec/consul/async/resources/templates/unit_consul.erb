Service Consul
<%
  service('consul').each do |service|
%><%=service['Node']['Node']%>:<%=service['Service']['Port']%>
<%
  end
%>