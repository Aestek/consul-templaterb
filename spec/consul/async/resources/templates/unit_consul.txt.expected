Service Consul
consul01-par.central.criteo.preprod:8300
consul02-par.central.criteo.preprod:8300
consul03-par.central.criteo.preprod:8300
